src/core/util/logger.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include <spdlog/sinks/stdout_color_sinks.h>
+#include <spdlog/spdlog.h>
+
+namespace util::logging {
+
+inline void Initialize(std::string const& logger_name = "desbordante",
+                       std::vector<spdlog::sink_ptr> sinks = {}) {
+    if (spdlog::get(logger_name)) {
```

Comment:
This part means the function only ensures that the logger is initialized, so name it `EnsureInitialized`.